group_vars/allsearch_frontend/production.yml

```
@@ -1,3 +1,4 @@
 ---
+allsearch_api_url: https://allsearch-api-production.princeton.edu
```

```suggestion
allsearch_api_url: https://allsearch-api.princeton.edu
```

The api url doesn't have `-production`

---

Thanks!